src/systemctl/systemctl.c

```diff
@@ -4268,9 +4294,15 @@ static void print_status_info(
                 next_rel_time = format_timestamp_relative(tstamp1, sizeof tstamp1, next_elapse);
                 next_time = format_timestamp(tstamp2, sizeof tstamp2, next_elapse);
 
-                if (next_time && next_rel_time)
+                if (next_time && next_rel_time) {
                         printf("%s; %s\n", next_time, next_rel_time);
-                else
+                        if (!strv_isempty(i->triggers)) {
+                                char **trigger;
+
+                                STRV_FOREACH(trigger, i->triggers)
+                                        printf("           %s %s\n", special_glyph(SPECIAL_GLYPH_TREE_RIGHT), *trigger);
```

Comment:
Hmm, I'm not sure if this is clear enough:
```console
$ build/systemctl status unbound-anchor.timer
● unbound-anchor.timer - daily update of the root trust anchor for DNSSEC
   Loaded: loaded (/usr/lib/systemd/system/unbound-anchor.timer; enabled; vendor preset: enabled)
   Active: active (waiting) since Mon 2019-09-16 10:02:49 CEST; 1s ago
  Trigger: Tue 2019-09-17 00:00:00 CEST; 13h left
           └─ unbound-anchor.service
     Docs: man:unbound-anchor(8)
```
Both the trigger time and the triggered units appear under the same heading. Wouldn't it be better to have a separate "Triggers: " section for the triggered units? Also, units that are triggered could be listed even if the triggering unit is stopped.

Reply:
@keur another update?